interm_notebook_5_classify.ipynb

```diff
@@ -0,0 +1,1175 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #3.    ticids = ['120461526', '278779899', '139754153', '273418879', '52121469', '188580272', '394015919', '402104884']
</pre><p> </p><p>Without increasing the runtime too much, could we run through a long list of these and only display graphs when the probability of a flare is high?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/spacetelescope/project-tikebook/pull/1/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='53'/>